### ISSUE TITLE
Demo: "Show Value Under Mouse, Multiple Series" is broken

### ISSUE THREAD

**Issue by Graat** [ScottPlot/ScottPlot#3680]
ScottPlot5 demo "Show Value Under Mouse, Multiple Series" responds unintuitively

<details>
<summary>suggested fix</summary>

https://github.com/ScottPlot/ScottPlot/blob/main/src/ScottPlot5/ScottPlot5%20Demos/ScottPlot5%20WinForms%20Demo/Demos/ShowValueOnHoverMultiple.cs

I added a line in the section where it is searching for the point which is nearest to the mouse position. This should fix the issue.
```cs
        formsPlot1.MouseMove += (s, e) =>
        {
            // determine where the mouse is
            Pixel mousePixel = new(e.Location.X, e.Location.Y);
            Coordinates mouseLocation = formsPlot1.Plot.GetCoordinates(mousePixel);

            // get the nearest point of each scatter
            Dictionary<int, DataPoint> nearestPoints = new();
            for (int i = 0; i < MyScatters.Count; i++)
            {
                DataPoint nearestPoint = MyScatters[i].Data.GetNearest(mouseLocation, formsPlot1.Plot.LastRender);
                nearestPoints.Add(i, nearestPoint);
            }

            // determine which scatter's nearest point is nearest to the mouse
            bool pointSelected = false;
            int scatterIndex = -1;
            double smallestDistance = double.MaxValue;
            for (int i = 0; i < nearestPoints.Count; i++)
            {
                if (nearestPoints[i].IsReal)
                {
                    // calculate the distance of the point to the mouse
                    double distance = nearestPoints[i].Coordinates.Distance(mouseLocation);
                    if (distance < smallestDistance)
                    {
                        // store the index
                        scatterIndex = i;
                        pointSelected = true;
                        smallestDistance = distance; //I added this line to update smallestDistance.
                    }
                }
            }

            // place the crosshair, marker and text over the selected point
            if (pointSelected)
            {
                ScottPlot.Plottables.Scatter scatter = MyScatters[scatterIndex];
                DataPoint point = nearestPoints[scatterIndex];

                MyCrosshair.IsVisible = true;
                MyCrosshair.Position = point.Coordinates;
                MyCrosshair.LineColor = scatter.MarkerStyle.Fill.Color;

                MyHighlightMarker.IsVisible = true;
                MyHighlightMarker.Location = point.Coordinates;
                MyHighlightMarker.MarkerStyle.Outline.Color = scatter.MarkerStyle.Fill.Color;

                MyHighlightText.IsVisible = true;
                MyHighlightText.Location = point.Coordinates;
                MyHighlightText.LabelText = $"{point.X:0.##}, {point.Y:0.##}";
                MyHighlightText.LabelFontColor = scatter.MarkerStyle.Fill.Color;

                formsPlot1.Refresh();
                base.Text = $"Selected Scatter={scatter.Label}, Index={point.Index}, X={point.X:0.##}, Y={point.Y:0.##}";
            }
```
</details>

**Comment by swharden** [comment]
Whoops, good catch!

This line:
```cs
smallestDistance = distance; 
```

should be inserted between lines 52 and 53:
https://github.com/ScottPlot/ScottPlot/blob/5ed3178403e56283dfd583b05890893d1d9ab2f0/src/ScottPlot5/ScottPlot5%20Demos/ScottPlot5%20WinForms%20Demo/Demos/ShowValueOnHoverMultiple.cs#L71-L85

If someone wants to open a pull request to correct this issue, I'd be happy to merge it in this evening!

**Comment by jamaa** [comment]
Good catch indeed! I was wondering why it was behaving weirdly sometimes, but couldn't figure out the cause. It's so obvious now :-) I made a PR #3684

**Comment by swharden** [comment]
resolved in #3684
